policies.go

```
@@ -353,7 +354,8 @@ func (r *roundRobinHostPolicy) AddHost(host *HostInfo) {
 }
 
 func (r *roundRobinHostPolicy) RemoveHost(host *HostInfo) {
-	r.hosts.remove(host.ConnectAddress())
+	connAddr, _ := host.ConnectAddress()
```

We can't ignore the error here, we're basically trying to remove an address that doesn't exist. Just return if there is an error

---

fixed